src/Entity/PageManager.php

```diff
@@ -35,10 +37,19 @@ final class PageManager extends BaseEntityManager implements PageManagerInterfac
     public function __construct(
         string $class,
         ManagerRegistry $registry,
-        private SlugifyInterface $slugify,
+        private SlugifyInterface|FixPageUrlInterface $fixPageUrl,
```

Comment:
why not changing this signature to
```
private SlugifyInterface|SluggerInterface
```
?

Either it make sens to introduce a FixPageUrl service and this should be done in a separate PR in which the name will be challenged, either it is not needed and we could simplify this pr